src/Interfaces.ts

```diff
@@ -1,72 +1,125 @@
-export interface Claim<T extends ClaimAttributes = ClaimAttributes> {
+import * as Joi from 'joi'
+import * as jsigs from 'jsonld-signatures'
+
+export enum ClaimType {
+  Identity = 'Identity',
+  Work = 'Work',
+}
+
+export interface ClaimAttributes {
```

Comment:
Question - shouldn't this disappear now?

Reply:
Pulling anything out of `object` is painful. So I was trying to think of a reason we would need to do that for the content of the claim(Attributes). I can't think of anything at the moment.